app/models/user.rb

```diff
@@ -53,7 +53,7 @@ class User < ActiveRecord::Base
 
   scope :admin, -> { where(is_admin: true) }
 
-  validates :email, presence: true
+  validates :email, :password, :password_confirmation, presence: true
```

Comment:
This should be done like:

``` Ruby
validates :email, presence: true

validates :password, confirmation: true
validates :password_confirmation, presence: true
```

You can read: http://guides.rubyonrails.org/active_record_validations.html#confirmation


with that you shouldn't need your last commit (acbc7df96a96807e5c893fe52a7573525f27ef55)